src/index.html

```diff
@@ -20,6 +20,10 @@
             properties: PROPERTIES,
             emailCaptureEnabled: true
         };
+        
+        Smooch.on('message', function(message) {
+          console.log('Message received', message);
```

Comment:
I was thinking about leaving it there since it's for debugging purpose and this whole page is for debugging purpose. Shows that it still works.